types/2020-08-27/Orders.d.ts

```diff
@@ -413,25 +413,9 @@ declare module 'stripe' {
                 requested_address_types?: Array<'zengin'>;
 
                 /**
-                 * The bank transfer type that this PaymentIntent is allowed to use for funding. Permitted values include: `us_bank_account`, `eu_bank_account`, `id_bank_account`, `gb_bank_account`, `jp_bank_account`, `mx_bank_account`, `eu_bank_transfer`, `gb_bank_transfer`, `id_bank_transfer`, `jp_bank_transfer`, `mx_bank_transfer`, or `us_bank_transfer`.
+                 * The bank transfer type that this PaymentIntent is allowed to use for funding Permitted values include: `jp_bank_transfer`.
                  */
-                type: BankTransfer.Type | null;
```

Comment:
maybe worth calling out in the changelog as in theory technically breaking but in fact just a better representation of the behavior that was allowed

Reply:
Done.